hyfer-backend/server/api/github.js

```diff
@@ -49,12 +49,11 @@ async function getTeamMembers(req, res) {
       created_at: teamsInfo[i].created_at,
       members: item,
     }));
-    res.send(modifiedTeamsStudents);
+    return modifiedTeamsStudents;
   } catch (error) {
-    console.log(res.statusCode);
+    console.log(error);
   }
 }
-
```

Comment:
Please add the blank line back in. Use a blank line between function definitions.